lib/yt/audit.rb

```diff
@@ -1,56 +1,36 @@
 require 'yt'
 require 'yt/annotations'
+require 'yt/video_audit/info_card'
+require 'yt/video_audit/brand_anchoring'
+require 'yt/video_audit/subscribe_annotation'
+require 'yt/video_audit/youtube_association'
+require 'yt/video_audit/end_card'
 
 module Yt
-  module Audit
-    # Audit any info card of a video
-    # @param [String] video_id the video to audit.
-    # @return [Boolean] if the video has any info card.
-    def self.has_info_cards?(video_id)
-      Yt::Annotations.for(video_id).any? do |annotation|
-        annotation.is_a? Yt::Annotations::Card
-      end
+  class Audit
+    def initialize(channel_id:, video_count: 10)
+      @channel_id = channel_id
+      @video_count = video_count
```

Comment:
I added `video_count` attribute here for cases so we can set it from audienceplus app.